blaze/compute/core.py

```diff
@@ -446,12 +446,13 @@ def compute(expr, d, **kwargs):
     optimize_ = kwargs.get('optimize', optimize)
     pre_compute_ = kwargs.get('pre_compute', pre_compute)
     post_compute_ = kwargs.get('post_compute', post_compute)
-
     expr2, d2 = swap_resources_into_scope(expr, d)
     if pre_compute_:
-        d3 = dict([(e, pre_compute_(expr2, dat, **kwargs))
```

Comment:
i'd like to leave the expression in the input for now and consider removing in release 0.8.2. @llllllllll go ahead and change the other `pre_compute_` call to be similar to this one